docs/docs/keymaps/input-processors/scaler.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Scaler Input Processor
+sidebar_label: Scaler
+---
+
+## Overview
+
+The scaler input processor is used to scale the value of an input event that has a code matching the codes set on the scaler. Events with other codes will be ignored. Values are scaled by multiplying by the multiplier parameter, and then dividing by the divisor parameter.
```

Comment:
Are there any overflow concerns? If so, is there a fixed max multiplier value that can be documented?

Reply:
So, we use a 16-bit integer for the math, and at most we are multiplying the event value by the multiplier before dividing.  Assuming relative event values are capped at `1024`, that leaves space for a multiplier of `64`, so I'd suggest we document a max of `16` to be extra cautious for the multiplier. Thoughts?

Reply:
Added that as a warning in the scaler docs, pending your thoughts on the matter.

Reply:
Looks good to me, thanks.